swerve_drive_controller/include/swerve_drive_controller/swerve_drive_controller.hpp

```diff
@@ -0,0 +1,192 @@
+// Copyright 2025 (your name or organization)
```

Comment:
please fill out, if you don't have an affiliation just add ros2_control development team